Packs/Rapid7AppSec/Integrations/Rapid7AppSec/Rapid7AppSec.yml

```diff
@@ -0,0 +1,737 @@
+category: Utilities
+commonfields:
+  id: rapid7appsec
+  version: -1
+configuration:
+- defaultvalue: https://xxx.api.insight.rapid7.com/
+  display: Server URL
+  name: url
+  required: true
+  type: 0
+- defaultvalue: 'false'
+  display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- defaultvalue: 'false'
+  display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: API Key
+  hiddenpassword: true
+  name: api_key
+  required: true
+  type: 9
+description: Rapid7 AppSec integration allows the management of applications vulnerabilities and scans.
+display: Rapid7AppSec
+name: rapid7appsec
+script:
+  commands:
+  - arguments:
+    - description: The ID of the Vulnerability (dependencies - use app-sec-vulnerability-list to get all vulnerability IDs).
+      isArray: false
+      name: vulnerability_id
+      required: true
+    - auto: PREDEFINED
+      description: The severity of the Vulnerability.
+      isArray: false
+      name: severity
+      predefined:
+      - Safe
+      - Informational
+      - Low
+      - Medium
+      - High
+      required: false
+    - auto: PREDEFINED
+      description: The status of the Vulnerability.
+      isArray: false
+      name: status
+      predefined:
+      - Unreviewed
+      - False Positive
+      - Verified
+      - Ignored
+      - Remediated
+      - Duplicate
+      required: false
+    description: Update the severity or the status of the vulnerability.
+    execution: false
+    name: app-sec-vulnerability-update
+  - arguments:
+    - description: The vulnerability ID to get. In case of using this argument, the pagination arguments not relevant.
```

Comment:
I don't write "dependencies - use app-sec-vulnerability-list to get all vulnerability IDs" because it should be the same command. If the user wants to get the IDs, he should run the command without the argument.